.cbmc-batch/include/cipher_openssl.h

```diff
@@ -0,0 +1,15 @@
+#include <aws/cryptosdk/private/cipher.h>
+
```

Comment:
Please add a comment saying why these are here.  They look like you are redefining structs that already exist in the sdk.

Reply:
These structs are from the SDK, but they are defined in the c files, and he needs them here to be able to access them from the harness.